README.md

```diff
@@ -1,26 +1,2 @@
 # blockchain-dev
 Studies on Distributed Computing, the Bitcoin Protocol and the Blockchain Network
-
-## Micropayments Proxy
-
```

Comment:
A proxy client, `http_proxy_client.py` will invoke a Bitcoin micropayments proxy server, `payments_handler.py`, when the client receives an HTTP status code 402 - `Payment Required` - from some service running on some server, `paywalled_server.py`, to which an HTTP request has been made. The payments proxy won't be activated under any other scenario. A plain vanilla server is included for testing purposes.